W/web3go/build_tarballs.jl

```diff
@@ -0,0 +1,47 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "web3go"
+version = v"0.2.89"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/gochain/web3/archive/refs/tags/v$(version).tar.gz", "4797b4bce370526da0fb04ca2350c36ac5d9894b1cb92a24524d2826ab0c34d6")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/web3-*/
+install_license LICENSE
+CGO_ENABLED=1 make
+mkdir -p $WORKSPACE/destdir/bin && mv web3* $WORKSPACE/destdir/bin
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("i686", "linux"; libc = "glibc"),
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("powerpc64le", "linux"; libc = "glibc"),
+    Platform("i686", "linux"; libc = "musl"),
+    Platform("x86_64", "linux"; libc = "musl"),
+    Platform("aarch64", "linux"; libc = "musl"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("x86_64", "freebsd"; ),
+    Platform("i686", "windows"; )
+]
```

Comment:
This is a bizarre set of platforms.  At a quick glance missing are macOS and 64-bit Windows?  I don't see any correlation between them.

Reply:
macOS and 64-bit Windows were excluded due to different compilation issues.